### PR TITLE
[training manual] GRASS chapter 13

### PR DESCRIPTION
**Pull request by ghtmtt**
Second PR for the Training Manual, GRASS integration, chapter 13:

* screenshot updates
* new sections and exercises
* better description of the *new* GRASS plugin

moreover, some icon addition in the `substitution.txt` file

### CODE REVIEW

File: source/docs/training_manual/grass/grass_tools.rst [hunk 1]
```diff
@@ -8,75 +8,120 @@
...
+:kbd:`g_aspect` layer from the :kbd:`g_dem` one. The value range gets from 0
+(North) passing through 90 (East), 180 (South), 270 (West) and finally to 360
+(North again). We can reclassify the :kbd:`g_aspect` layer to have just 4
+**categories** following specific *rules* (North = 1, East = 2, South = 3 and
+West = 4).
+
+Grass reclassify tool accepts a ``txt`` file with the defined rules. Writing the
+rules is very simple and the GRASS Manual contains very good description.
+
+.. tip:: each GRASS tool has its own Manual tab. Take the time to read the
```
Comment (yjacolin) [inline source/docs/training_manual/grass/grass_tools.rst]: **E**ach?

File: source/docs/training_manual/grass/grass_tools.rst [hunk 2]
```diff
@@ -8,75 +8,120 @@
...
+  description of the tool you are using to don't miss some useful parameter
```
Comment (yjacolin) [inline source/docs/training_manual/grass/grass_tools.rst]: parameter**s**?

File: source/docs/training_manual/grass/grass_tools.rst [hunk 3]
```diff
@@ -8,75 +8,120 @@
...
+manage and to process.
 
-.. image:: img/nviz_result.png
+.. image:: img/grass_reclass_result.png
    :align: center
 
-Experiment with the :guilabel:`height`, :guilabel:`z-exag`, and :guilabel:`View
-method` settings to change your view of the data. The navigation methods may
-take some getting used to.
+.. tip:: open the :kbd:`reclass_aspect.txt` with a text editor to see the rules
```
Comment (yjacolin) [inline source/docs/training_manual/grass/grass_tools.rst]: **O**pen?

File: source/docs/training_manual/grass/grass_tools.rst [hunk 4]
```diff
@@ -89,36 +134,38 @@ so:
...
 
-* Click :guilabel:`View output` to see the output displayed in your map:
-
-  .. image:: img/altitude_range_output.png
-     :align: center
+This shows all the areas where the terrain higher than 1000 meters.
 
-This shows all the areas where the terrain is lower than 500 meters or higher
-than 1000 meters.
+.. tip:: you can also save the formula you have created and load it in another
```
Comment (yjacolin) [inline source/docs/training_manual/grass/grass_tools.rst]: **Y**ou

File: source/docs/training_manual/grass/grass_setup.rst [hunk 5]
```diff
@@ -72,42 +104,95 @@ geographic area you'll be working in.
...
+load the data in QGIS is the Browser Panel.
 
-  .. image:: img/grass_tool_selected.png
+GRASS data are recognized from the QGIS Browser as *real* GRASS data and you can
+noticed it because you will see the GRASS icon next to the GRASS Mapset.
+
+.. image:: img/grass_browser.png
+   :align: center
+
+.. note:: moreover you will see the |grassMapsetOpen| icon next to the Mapset
```
Comment (yjacolin) [inline source/docs/training_manual/grass/grass_setup.rst]: **M**oreover

File: source/docs/training_manual/grass/grass_setup.rst [hunk 6]
```diff
@@ -72,42 +104,95 @@ geographic area you'll be working in.
...
+  that is opened
+
+.. warning:: you will see a replication of the GRASS Location as normal folder:
```
Comment (yjacolin) [inline source/docs/training_manual/grass/grass_setup.rst]: **Y**ou

File: source/docs/training_manual/grass/grass_setup.rst [hunk 7]
```diff
@@ -72,42 +104,95 @@ geographic area you'll be working in.
...
+* Open the :kbd:`shapefiles` folder and simply drag the :kbd:`roads.shp` layer
+  into the ``grass_mapset`` Mapset.
+
+  .. image:: img/grass_browser_import.png
+     :align: center
+
+That's it! If you expand the Mapset you will see the imported :kbd:`roads.shp`
+layer. You can now load in QGIS the imported layer like all the other layers.
+
+.. tip:: you can also load layers from the Layer Legend Panel to Mapset in the
```
Comment (yjacolin) [inline source/docs/training_manual/grass/grass_setup.rst]: **Y**ou

File: source/docs/training_manual/grass/grass_setup.rst [hunk 8]
```diff
@@ -72,42 +104,95 @@ geographic area you'll be working in.
...
+
+We will use now the *long* method to load the :kbd:`rivers.shp` layer into the
+same Mapset.
+
+* Load data into QGIS as usual. Use the :kbd:`rivers.shp` dataset (found in the
+  :kbd:`exercise_data/shapefiles` folder)
+* As soon as it is loaded, click on the :guilabel:`Filter` box of the ``GRASS Panel``
+  and find the vector import tool by entering the term :kbd:`v.in.ogr.qgis`:
+
+  .. warning:: there are 2 similar tools: :kbd:`v.in.ogr.qgis` and
```
Comment (yjacolin) [inline source/docs/training_manual/grass/grass_setup.rst]: **T**here

File: source/docs/training_manual/grass/grass_setup.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (ghtmtt) [reply source/docs/training_manual/grass/grass_setup.rst]: I changed this with a note tag, more appropriate

File: source/docs/training_manual/grass/grass_setup.rst [hunk 9]
```diff
@@ -72,42 +104,95 @@ geographic area you'll be working in.
...
 
-  .. image:: img/grass_tool_selected.png
+GRASS data are recognized from the QGIS Browser as *real* GRASS data and you can
+noticed it because you will see the GRASS icon next to the GRASS Mapset.
+
+.. image:: img/grass_browser.png
+   :align: center
+
+.. note:: moreover you will see the |grassMapsetOpen| icon next to the Mapset
+  that is opened
```
[the remark this answers is not in the document]
Comment (ghtmtt) [reply source/docs/training_manual/grass/grass_setup.rst]: better yes.. done

File: source/docs/training_manual/grass/grass_setup.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (ghtmtt) [reply source/docs/training_manual/grass/grass_setup.rst]: sure about that? If you agree I will skip a title for a simple tip tag.. agreed for more *complicated*

File: source/docs/training_manual/grass/grass_setup.rst [hunk 10]
```diff
@@ -162,10 +237,74 @@ Now that it's reprojected, you can load it into your GRASS database.
...
+on the Mapset (remember, the Mapset has the |grass| GRASS icon next to it). You
+will see some option: click on :kbd:`Open mapset`:
+
+.. image:: img/grass_open_mapset_browser.png
+   :align: center
+
+The Mapset is now open and ready to use!
+
+
+.. tip:: right click on a GRASS Mapset offers you a lot of different settings. Try
```
[the remark this answers is not in the document]
Comment (ghtmtt) [reply source/docs/training_manual/grass/grass_setup.rst]: I think a tip tag is better here. ;)

**Comment by ghtmtt** [comment]
@yjacolin @DelazJ I'm going to merge this so that we can resolve some conflicts with the answer chapter step by step. Thanks guys!

**Pull request merged**